SmartDeviceLinkTests/RPCSpecs/StructSpecs/SDLDisplayCapabilitiesSpec.m

```diff
@@ -49,18 +52,21 @@
         expect(testStruct.templatesAvailable).to(equal([@[@"String", @"String", @"String"] mutableCopy]));
         expect(testStruct.screenParams).to(equal(screenParams));
         expect(testStruct.numCustomPresetsAvailable).to(equal(@43));
+        expect(testStruct.menuLayoutsAvailable).to(equal(testLayout));
     });
     
     it(@"Should get correctly when initialized", ^ {
-        NSMutableDictionary* dict = [@{SDLRPCParameterNameDisplayType:SDLDisplayTypeGen26DMA,
+        NSMutableDictionary* dict = @{SDLRPCParameterNameDisplayType:SDLDisplayTypeGen26DMA,
                                        SDLRPCParameterNameDisplayName: @"test",
                                        SDLRPCParameterNameTextFields:[@[textField] mutableCopy],
                                        SDLRPCParameterNameImageFields:[@[imageField] mutableCopy],
                                        SDLRPCParameterNameMediaClockFormats:[@[SDLMediaClockFormatClockText1, SDLMediaClockFormatClock3, SDLMediaClockFormatClockText3] copy],
                                        SDLRPCParameterNameGraphicSupported:@YES,
                                        SDLRPCParameterNameTemplatesAvailable:[@[@"String", @"String", @"String"] mutableCopy],
                                        SDLRPCParameterNameScreenParams:screenParams,
-                                       SDLRPCParameterNameNumberCustomPresetsAvailable:@43} mutableCopy];
+                                       SDLRPCParameterNameNumberCustomPresetsAvailable:@43,
+                                      SDLRPCParameterNameMenuLayoutsAvailable: testLayout
```

Comment:
This should be removed since MenuLayoutsAvailable is no longer part of `SDLDisplayCapabilities`